GettingStarted/HelloWorld.ipynb

```diff
@@ -374,7 +310,10 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "%%bash\n",
```

Comment:
@Madejski Check this out! Merlin didn't know how to do this, so I guess its useful for everyone :-)